install/lsstsw.rst

```diff
@@ -98,9 +98,10 @@ Once the LSST Science Pipelines are installed, you can verify that it works by :
 You can upgrade an lsstsw installation in-place by running the following commands from within your :file:`lsstsw/` directory.
 
 1. `git pull` to download the latest environment definition.
-2. `./bin/deploy` to install that new conda environment.
+2. `bin/deploy` to install that new conda environment.
 3. Start a new shell for the final command, to ensure your shell environment is properly configured for the new lsstsw env, and source ``lsstsw/bin/envconfig`` if it is not automatically sourced during your shell startup.
-4. `rebuild -u lsst_distrib` to download the latest repos definition file and rebuild the entire Science Pipelines codebase.
+4. `bin/deploy` again to fully build the new conda environment.
```

Comment:
This should definitely not be necessary.  Can you describe what happened when you didn't do this?

Reply:
I've also never had to do this.

Reply:
After the first `bin/deploy` I started a new shell, and got this:

```
$ rebuild -u lsst_distrib
Environment lsst-scipipe-0.4.1 not active.
Please activate the environment using bin/envconfig before running rebuild.

$ bin/envconfig
Setting the environment... lsst-scipipe-0.4.1
Could not find conda environment: lsst-scipipe-0.4.1
You can list all discoverable environments with `conda info --envs`.

EUPS_PATH set to /Users/parejkoj/lsst/lsstsw/stack/0.4.1
notice: lsstsw tools have been set up.

$ rebuild -u lsst_distrib
Environment lsst-scipipe-0.4.1 not active.
Please activate the environment using bin/envconfig before running rebuild.
```

When I then ran `bin/deploy` in that second shell, it did a lot more than the first time in the previous shell.

Note that this was going from cb4e2dc to 0.4.1, if that matters.

Reply:
I think if you run deploy from an already active environment it doesn't install the new env but just checks your current env.

Reply:
Ah, that is an important piece of information. I source envconfig in my `.bashrc`, so I'm always in an active environment. So I guess what's happening here is that when I run `deploy` it breaks the existing environment so that when I start a new shell there isn't one, so the second `deploy` works?

So step zero of this list should be "start a shell without an activated Rubin environment"?

Reply:
Yes. I think that is right.

Reply:
I've replaced step 1. and made the list auto-enumerated. How's that?